inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/AnnotateVcf.SingleBatch.json.tmpl

```diff
@@ -1,6 +1,6 @@
 {
-  "AnnotateVcf.vcf" : "${this.cleaned_vcf}",
-  "AnnotateVcf.vcf_idx" : "${this.cleaned_vcf_index}",
+  "AnnotateVcf.vcf_list" : "${this.cleaned_vcf}",
+  "AnnotateVcf.vcf_idx_list" : "${this.cleaned_vcf_index}",
```

Comment:
Are these array types in the workspace?

Reply:
No, but Terra is able to handle conversion from a File to an Array[File] in inputs